### PR TITLE
Feat/389053 ticket transfer layout

### PR DESCRIPTION
**Pull request by leonardogbr**

### CODE REVIEW

File: sample/lib/main.dart [hunk 1]
```diff
@@ -51,33 +51,33 @@ class HomePage extends StatelessWidget {
...
-            const Divider(color: DSColors.neutralDarkCity),
-            const SampleToastShowcase(),
-            const Divider(color: DSColors.neutralDarkCity),
-            SampleRadioShowcase(),
-            const Divider(color: DSColors.neutralDarkCity),
-            SampleGroupCardShowcase(),
-            const Divider(color: DSColors.neutralDarkCity),
-            const SampleHeaderShowcase(),
-            const Divider(color: DSColors.neutralDarkCity),
+            // SampleMessageBubbleShowcase(),
```
Comment (mpamaro) [inline sample/lib/main.dart:54]: You should uncomment the other widgets.

**Review by mpamaro — APPROVED** [review]

**Comment by githubdoandre** [comment]
golden tests failed

File: lib/src/widgets/radio/ds_radio_tile.widget.dart [hunk 2]
```diff
@@ -41,23 +45,41 @@ class DSRadioTile<T> extends StatelessWidget {
...
+          children: [
+            DSRadio(
+              isEnabled: isEnabled,
+              value: value,
+              groupValue: groupValue,
+              onChanged: onChanged,
+              materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+            ),
+            const SizedBox(
+              width: 8,
```
Comment (githubdoandre) [inline lib/src/widgets/radio/ds_radio_tile.widget.dart:66]: use decimals, `8.0`

File: lib/src/widgets/radio/ds_radio_tile.widget.dart [hunk 3]
```diff
@@ -41,23 +45,41 @@ class DSRadioTile<T> extends StatelessWidget {
...
+            ),
+            Flexible(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  title,
+                  const SizedBox(
+                    height: 5,
```
Comment (githubdoandre) [inline lib/src/widgets/radio/ds_radio_tile.widget.dart:74]: same above

**Review by githubdoandre — APPROVED** [review]

**Pull request merged**